MAUI/ProgressBar/migration.md

```diff
@@ -0,0 +1,725 @@
+<td>Gets or sets the brush that paints the interior area of the track.</td></tr>
+<tr>
+<td>{{'[SegmentCount](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_SegmentCount)'| markdownify }}</td>
+<td>{{'SegmentCount'| markdownify }}</td>
+<td>Gets or sets the value that determine the segments count of progress bar.</td></tr>
+<tr>
+<td>{{'[GapWidth](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_GapWidth)'| markdownify }}</td>
+<td>{{'SegmentGapWidth'| markdownify }}</td>
+<td>Gets or sets the value that determines the gap between the segments.</td></tr>
+<tr>
+<td>{{'[RangeColors](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_RangeColors)'| markdownify }}</td>
+<td>{{'GradientStops'| markdownify }}</td>
+<td>Gets or sets a collection of ProgressGradientStop to fill the gradient brush to the progress.</td></tr>
+<tr>
+<td>{{'[AnimationDuration](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_AnimationDuration)'| markdownify }}</td>
+<td>{{'AnimationDuration'| markdownify }}</td>
+<td>Gets or sets a value that specifies the progress animation duration in milliseconds.</td></tr>
+<tr>
+<td>{{'[IndeterminateAnimationDuration](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_IndeterminateAnimationDuration)'| markdownify }}</td>
+<td>{{'IndeterminateAnimationDuration'| markdownify }}</td>
+<td>Gets or sets a value that specifies the indeterminate animation duration in milliseconds.</td></tr>
+<tr>
+<td>{{'[EasingEffect](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_EasingEffect)'| markdownify }}</td>
+<td>{{'AnimationEasing'| markdownify }}</td>
+<td>Gets or sets a value that specifies the easing effect for progress animation.</td></tr>
+<tr>
+<td>{{'[IndeterminateEasingEffect](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_IndeterminateEasingEffect)'| markdownify }}</td>
+<td>{{'IndeterminateAnimationEasing'| markdownify }}</td>
+<td>Gets or sets a value that specifies the easing effect for indeterminate animation.</td></tr>
+<tr>
+<td>{{'[IndeterminateIndicatorWidth](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_IndeterminateIndicatorWidth)'| markdownify }}</td>
+<td>{{'IndeterminateIndicatorWidthFactor'| markdownify }}</td>
+<td>Gets or sets the value that specifies width of the indeterminate indicator.</td></tr>
+</table> 
+
+<table>
```

Comment:
Before adding the code snippet, provide the below sentence

The following code example explains how to initialize the Xamarin progress bar and the .NET MAUI progress bar along with common properties.

Reply:
done